pkg/controller/sns/topic/hooks_test.go

```diff
@@ -0,0 +1,148 @@
+/*
+Copyright 2020 The Crossplane Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package topic
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	svcsdk "github.com/aws/aws-sdk-go/service/sns"
+
+	"github.com/crossplane/provider-aws/apis/sns/v1alpha1"
+	aws "github.com/crossplane/provider-aws/pkg/clients"
```

Comment:
```suggestion
	"testing"

	"github.com/google/go-cmp/cmp"
	svcsdk "github.com/aws/aws-sdk-go/service/sns"

	"github.com/crossplane/provider-aws/apis/sns/v1alpha1"
	aws "github.com/crossplane/provider-aws/pkg/clients"
```
Another minor nitpick